runtime/libffi/z/ffi64.c

```diff
@@ -764,6 +766,50 @@ ffi_prep_args (unsigned char *stack, extended_cif *ecif)
 
 }
 
+/**
+ * Helper functions to know if the given struct needed to be treated for Complex
+ * type for Float or Doubles
```

Comment:
This sentence should end with a period; "needed" should be "needs"; and `complex`, `float` and `double` should not be capitalized.

Reply:
Fixed in https://github.com/eclipse-openj9/openj9/pull/19645/commits/38245ec2872c2b2664f6fdd3947f80e71e63c57d#diff-7e388e6ac3235f9470ab3d7607ffbe1c8fceacddd69ac992fdae6cda3e99257d